app/webpacker/styles/links-and-buttons.scss

```diff
@@ -41,6 +41,7 @@ a {
   @include button;
   @include chevron;
   display: inline-block;
+  cursor: pointer;
```

Comment:
Not so sure about this bit. A link in a green box is still a link.

Reply:
According to the CSS spec a link is when we _should_ use a pointer type cursor; I don't think this line is actually necessary as we do this already (I added it to be explicit, but anchor tags default to pointer cursors). It's when we have a non-link button that DAC tell us to go against the standards and also use a pointer type cursor 🤷‍♂️

Reply:
Ah sorry - I got muddled between pointer and default (arrow).

I think it's because when it's a link people have extra options - middle click to open in a new tab, right click to bookmark, drag to desktop etc. With a button/`input[type='submit']` they don't, but if we use the same cursor there's no visual distinction.